Klarna.php

```diff
@@ -4417,6 +4418,19 @@ private function _checkConfig($config = null)
         }
     }
 
+    /**
+     * Fixes originalMessage encoding
+     *
+     * @param string $originalMessage
+     * @return string
+     */
+    private function fixMessageEncoding($originalMessage)
+    {
+        $possibleInputEncodings = ['ISO-8859-1', 'UTF-8'];
+        $outputEncoding = mb_internal_encoding();
```

Comment:
> mbstring is a non-default extension. This means it is not enabled by default. 

Probably need something like:

```php
if (!extension_loaded('mbstring')) {
    return $originalMessage;
}

...
```

As encodings in PHP is a bit messy, I don't know if using `iconv` or the `utf8_decode`/`utf8_encode` might be a more generic solution that everyone can benefit from.